.github/label-commenter-config.yml

```diff
@@ -0,0 +1,14 @@
+comment:
+  header: A brief note
+  footer: "\
+    ---\n\n
+    > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](http://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile: on the project.
+    "
+
+labels:
```

Comment:
@amino19 This looks good.
Let's also add comments for other labels? Namely `issue/design` and `issue/dco`?

Reply:
@DelusionalOptimist: Just done! :slightly_smiling_face: